src/blueprints/timeline-state-resolver-types/caspar-cg-types.ts

```diff
@@ -12,6 +12,17 @@ export interface CasparCgMediaTimelineObject extends TimelineObject {
     length?: number
     playing?: boolean
     noStarttime?: boolean // The typo is used by TSR... :(
+    mixer?: {
+      keyer?: boolean
```

Comment:
It's optional on the type used/declared in TSR, which this is copy of.
You don't always need to set the `keyer` when using the `mixer`. It just looks like that because that is currently our only use-case of the `mixer`.